video/out/w32_common.c

```diff
@@ -953,6 +953,28 @@ static bool is_visible(HWND window)
     return GetWindowLongPtrW(window, GWL_STYLE) & WS_VISIBLE;
 }
 
+static bool update_transparency(struct vo_w32_state *w32, bool handleDefault) {
+    if (!w32 || w32->parent) {
+        return false;
+    }
+
+    int transparency = w32->opts->window_transparency;
+
+    if (transparency == -1 && handleDefault) { // Reset the window attributes
+        // Remove the WS_EX_LAYERED flag from the extended style and set the alpha value to 255
+        return SetWindowLong(w32->window, GWL_EXSTYLE, GetWindowLong(w32->window, GWL_EXSTYLE) & ~WS_EX_LAYERED)
+            && SetLayeredWindowAttributes(w32->window, 0, 255, LWA_ALPHA);
+    } else if (transparency <= -1 && !handleDefault) {
+        return false;
+    } else { // Set the window transparency
+        // Clamp the transparency value to 0 or 255 if it exceeds the minimum or maximum, respectively
+        int alpha = transparency < 0 ? 0 : (transparency > 255 ? 255 : transparency);
```

Comment:
MP_MAX and MP_MIN are more readable.

Reply:
I will change it later once I get to my pc.